### PR TITLE
feat(connector): [Bluesnap] Remove wallet call 

### PR DESCRIPTION
**Pull request by Sangamesh26**
## Type of Change

- [x] New feature

## Description

Remove Wallet call form bluesnap which was earlier treated a session call.
We get session object now from apple only instead of bluesnap.

## Motivation and Context

Get session response from Apple pay instead of getting from connector

## How did you test it?

Manual

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed submitted code

### CODE REVIEW

**Review by SanchithHegde — DISMISSED** [review]
@Sangamesh26 Please attach suitable labels

File: crates/router/src/connector/bluesnap.rs [hunk 1]
```diff
@@ -504,84 +504,6 @@ impl api::PaymentSession for Bluesnap {}
 impl ConnectorIntegration<api::Session, types::PaymentsSessionData, types::PaymentsResponseData>
     for Bluesnap
 {
-    fn get_headers(
```
Comment (ArjunKarthik) [inline crates/router/src/connector/bluesnap.rs:507]: @Sangamesh26 Do not remove this code, let it be and add comments. Revert only the core changes

File: crates/router/src/connector/bluesnap/transformers.rs [hunk 2]
```diff
@@ -299,96 +286,6 @@ impl From<api_models::payments::ApplepayPaymentMethod> for ApplepayPaymentMethod
     }
 }
 
-impl TryFrom<&types::PaymentsSessionRouterData> for BluesnapCreateWalletToken {
```
Comment (ArjunKarthik) [inline crates/router/src/connector/bluesnap/transformers.rs:302]: Same here.

**Review by ArjunKarthik — APPROVED** [review]

**Review by ArjunKarthik — CHANGES_REQUESTED** [review]
Let's not remove the code for session call

**Comment by ArjunKarthik** [comment]
@Sangamesh26 Please add necessary labels to the PR

**Review by ArjunKarthik — APPROVED** [review]

**Review by Narayanbhat166 — APPROVED** [review]

**Pull request merged**